src/plugins/waitlist.js

```diff
@@ -31,14 +31,33 @@ export default function waitlistPlugin () {
       }
     }
 
-    function onModAddDj () {
-      // TODO
+    function onModAddDj (ref) {
+      mp.emit('modAddDj', {
+        mod: mp.user(ref.mi) || mp.wrapUser({ id: ref.mi }),
```

Comment:
I'd prefer to call this property `moderator` (same in the other events). miniplug doesn't really shorten words anywhere else